pkg/agent/agent.go

```diff
@@ -730,6 +730,11 @@ func (i *Initializer) setupGatewayInterface() error {
 	if err := i.setInterfaceMTU(i.hostGateway, i.networkConfig.InterfaceMTU); err != nil {
 		return err
 	}
+	if i.nodeConfig.GatewayConfig.IPv4 != nil {
+		if err := i.setInterfaceARPAnnounce(gatewayIface.InterfaceName, 1); err != nil {
```

Comment:
Please add comment why this is needed

Reply:
Added comment and PR description about this context.